packages/lib/.storybook/main.ts

```diff
@@ -11,14 +11,7 @@ export default {
     const { mergeConfig } = await import("vite");
 
     return mergeConfig(config, {
-      plugins: [
-        tsconfigPaths(),
-        {
-          react: {
-            jsxImportSource: "@emotion/react",
-          },
-        },
-      ],
```

Comment:
Is this no longer necessary?

Reply:
it's not necessary since we don't use the css prop in any stories in lib